src/urbit/app/chess.hoon

```diff
@@ -213,6 +228,39 @@
           %=  this
             games  (~(put by games) game-id.action u.game(got-draw-offer |))
           ==
+        %claim-special-draw
+          =/  game-state  (~(get by games) game-id.action)
+          ?~  game-state
+            :_  this
+            =/  err
+              "no active game with id {<game-id.action>}"
+            :~  [%give %poke-ack `~[leaf+err]]
+            ==
+          ?.  (check-threefold fen-repetition.game-state position.game-state)
+            :_  this
+            =/  err
+              "no special draw available for game {<game-id.action>}"
+              :~  [%give %poke-ack `~[leaf+err]]
+              ==
+          :-  :~  :*  %give  %fact  ~[/game/(scot %da game-id.action)/moves]
+                      %chess-result  !>([game-id.action %'½–½' ~])
+                  ==
+                  :*  %give  %fact  ~[/game/(scot %da game-id.action)/updates]
+                      %chess-update
+                      !>([%result game-id.action %'½–½'])
+                  ==
+                  :*  %give  %kick  :~  /game/(scot %da game-id.action)/updates
+                                        /game/(scot %da game-id.action)/moves
+                                    ==
+                      ~
+                  ==
+              ==
+          =/  updated-game  game.u.game-state
+          =.  result.updated-game  `(unit chess-result)``%'½–½'
+          %=  this
+            games    (~(del by games) game-id.action)
+            archive  (~(put by archive) game-id.action updated-game)
+          ==
```

Comment:
Without having tested yet this poke looks okay, but why refer to it as a `special-draw` and not a `threefold-draw` when the poke is just checking for a threefold repetition? If I'm not missing something (I've read the previous discussion in this PR, but I might be) I think it would be clearer if it was called something like `threefold-draw` throughout.

Reply:
Per [this comment](https://github.com/ashelkovnykov/urbit-chess/pull/40#discussion_r980612639), the same pokes and settings will eventually be used for the 50 move rule as well.

Reply:
Yes, I asked for this change so that we could handle both threefold repetition and the 50-move rule with the same code